diff --git a/dist/index.mjs b/dist/index.mjs
index f6441a1..4d15bd7 100644
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -12,7 +12,6 @@ import pluginComments from "@eslint-community/eslint-plugin-eslint-comments";
 import pluginImportLite from "eslint-plugin-import-lite";
 import pluginNode from "eslint-plugin-n";
 import pluginPerfectionist from "eslint-plugin-perfectionist";
-import pluginUnicorn from "eslint-plugin-unicorn";
 import pluginUnusedImports from "eslint-plugin-unused-imports";
 import globals from "globals";
 import { mergeProcessors, processorPassThrough } from "eslint-merge-processors";
@@ -2405,7 +2404,6 @@ function antfu(options = {}, ...userConfigs) {
 		isInEditor,
 		...enableE18e === true ? {} : enableE18e
 	}));
-	if (enableUnicorn) configs.push(unicorn(enableUnicorn === true ? {} : enableUnicorn));
 	if (enableVue) componentExts.push("vue");
 	if (enableJsx) configs.push(jsx(enableJsx === true ? {} : enableJsx));
 	if (enableTypeScript) configs.push(typescript({
diff --git a/package.json b/package.json
index 638880e..77ff13e 100644
--- a/package.json
+++ b/package.json
@@ -134,7 +134,6 @@
     "eslint-plugin-pnpm": "^1.7.0",
     "eslint-plugin-regexp": "^3.1.1",
     "eslint-plugin-toml": "^1.5.0",
-    "eslint-plugin-unicorn": "^73.0.0",
     "eslint-plugin-unused-imports": "^4.4.1",
     "eslint-plugin-vue": "^10.10.0",
     "eslint-plugin-yml": "^3.8.1",
