diff --git a/dist/index.mjs b/dist/index.mjs
index 4fee2be6b8df63bb71594290a2e113b40aaeed60..9493a1368ecbdd38cfbb3bf88ea918eb4fbd94e4 100644
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -12,7 +12,6 @@ import pluginAntfu from "eslint-plugin-antfu";
 import pluginImportLite from "eslint-plugin-import-lite";
 import pluginNode from "eslint-plugin-n";
 import pluginPerfectionist from "eslint-plugin-perfectionist";
-import pluginUnicorn from "eslint-plugin-unicorn";
 import pluginUnusedImports from "eslint-plugin-unused-imports";
 import globals from "globals";
 import { mergeProcessors, processorPassThrough } from "eslint-merge-processors";
@@ -2004,7 +2003,7 @@ async function typescript(options = {}) {
 		}] : [],
 		...erasableOnly ? [{
 			name: "antfu/typescript/erasable-syntax-only",
-			plugins: { "erasable-syntax-only": await interopDefault(import("./lib-BGCOhS6c.mjs")) },
+			plugins: { "erasable-syntax-only": await interopDefault(import("eslint-plugin-erasable-syntax-only")) },
 			rules: {
 				"erasable-syntax-only/enums": "error",
 				"erasable-syntax-only/import-aliases": "error",
@@ -2405,7 +2404,6 @@ function antfu(options = {}, ...userConfigs) {
 		isInEditor,
 		...enableE18e === true ? {} : enableE18e
 	}));
-	if (enableUnicorn) configs.push(unicorn(enableUnicorn === true ? {} : enableUnicorn));
 	if (enableVue) componentExts.push("vue");
 	if (enableJsx) configs.push(jsx(enableJsx === true ? {} : enableJsx));
 	if (enableTypeScript) configs.push(typescript({
diff --git a/package.json b/package.json
index 2b2c08f43e9ce9678143fce3441a04414ea81bdd..d9ba31304c6fa394d4aec69089cf05c26eeb76a7 100644
--- a/package.json
+++ b/package.json
@@ -130,7 +130,6 @@
     "eslint-plugin-pnpm": "^1.6.1",
     "eslint-plugin-regexp": "^3.1.1",
     "eslint-plugin-toml": "^1.4.0",
-    "eslint-plugin-unicorn": "^72.0.0",
     "eslint-plugin-unused-imports": "^4.4.1",
     "eslint-plugin-vue": "^10.10.0",
     "eslint-plugin-yml": "^3.6.0",
